config/routes/sessions.js

```diff
@@ -13,17 +13,6 @@ module.exports.routes = {
     }
   },
 
-  'GET /en/sessions/:session/download': {
-    name: 'sessions.download',
-    controller: 'SessionsController',
-    action: 'download',
-    lang: 'en',
-    i18n: {
-      en: '/en/sessions/:session/download',
-      fr: '/fr/sessions/:session/download'
-    }
-  },
-
   'GET /en/sessions/:session/view': {
```

Comment:
TBD: we should probably change the route path (and controller name?)

Reply:
Agreed we should add it to the backlog